src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -19,8 +19,8 @@ public class PersonBuilder {
     public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_ID = "P24689";
     public static final String DEFAULT_WARD = "A1";
-    public static final String DEFAULT_DIAGNOSIS = "Celiac Disease";
-    public static final String DEFAULT_MEDICATION = "gluten-free diet";
+    public static final String DEFAULT_DIAGNOSIS = "";
+    public static final String DEFAULT_MEDICATION = "";
```

Comment:
could have maybe created a different diagnosis / medication string that is meant to be empty, as this might cause issues with other test cases

Reply:
That is possible, but I think it is unlikely as the `withSomething()` method is usually chained on to this PersonBuilder constructor for those cases where the parameters are specified.